src/main/java/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariable.java

```diff
@@ -0,0 +1,148 @@
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins.ownership.model.workflow;
+
+import com.synopsys.arc.jenkins.plugins.ownership.OwnershipDescription;
+import com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerHelper;
+import com.synopsys.arc.jenkins.plugins.ownership.nodes.NodeOwnerHelper;
+import groovy.lang.Binding;
+import hudson.Extension;
+import hudson.model.Node;
+import hudson.model.Run;
+import java.io.IOException;
+import java.io.InputStream;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import jenkins.model.Jenkins;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.ProxyWhitelist;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.StaticWhitelist;
+import org.jenkinsci.plugins.workflow.cps.CpsScript;
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable;
+import org.jenkinsci.plugins.workflow.support.steps.build.RunWrapper;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * Provides {@link OwnershipDescription}s for different item types.
+ * @author Oleg Nenashev
+ * @since TODO
+ */
+@Extension
+public class OwnershipGlobalVariable extends GlobalVariable {
+
+    @Override 
+    public final Object getValue(CpsScript script) throws Exception {   
+        final Binding binding = script.getBinding();
+        final Object loadedObject;
+        if (binding.hasVariable(getName())) {
+            loadedObject = binding.getVariable(getName());
+        } else {
+            // Note that if this were a method rather than a constructor, we would need to mark it @NonCPS lest it throw CpsCallableInvocation.
+            loadedObject = script.getClass().getClassLoader()
+                    .loadClass(getClassName())
+                    .getConstructor(CpsScript.class).newInstance(script);
+            binding.setVariable(getName(), loadedObject);
+        }
+        return loadedObject;
+    }
+    
+    /**
+     * Canonical name of the class to be loaded.
+     * @return Canonical class name
+     */
+    @Nonnull
+    private String getClassName() {
+        return getClass().getName() + ".Impl";
+    }
+    
+    @Override 
+    public String getName() {
+        return "ownership";
+    }
+    
+    /**
+     * Loads a snippet from the resource file.
+     * @param name Name of the snippet (e.g. &quot;loadMultipleFiles&quot;)
+     * @return Workflow script text
+     * @throws IOException Loading error
+     */
+    @Restricted(NoExternalUse.class)
+    public static String getSampleSnippet(String name) throws IOException {
```

Comment:
It will be used for proposing Pipeline samples once Pipeline plugin provides such extension point. I've marked the method as restricted, so we will be able to change the implementation later